Unable to build site locally

I've attempted to build the site locally through various methods, including via the `Makefile` and with `jupyter-book`.  So far, I haven't been successful with either approach. When I try to build with the `Makefile` (after manually installing the dependencies to work around #63) I see the following:

```
make[1]: *** book: No such file or directory.  Stop.
make: *** [Makefile:37: html] Error 2
```

I tried working through some of the other steps in the Makefile manually, but haven't been able to progress to getting the site built.

When I try with jupyter-book (e.g. `jb build .`) I see the following:

<details>
  <summary>jb build output</summary>
<pre>
Running Jupyter-Book v0.11.3
Traceback (most recent call last):
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/jupyter_book/cli/main.py", line 244, in build
    parse_toc_yaml(toc)
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/sphinx_external_toc/parsing.py", line 82, in parse_toc_yaml
    return parse_toc_data(data)
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/sphinx_external_toc/parsing.py", line 88, in parse_toc_data
    raise MalformedError(f"toc is not a mapping: {type(data)}")
sphinx_external_toc.parsing.MalformedError: toc is not a mapping: <class 'list'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ross/.virtualenvs/skimage-tutorials/bin/jb", line 8, in <module>
    sys.exit(main())
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/jupyter_book/cli/main.py", line 246, in build
    _error(
  File "/home/ross/.virtualenvs/skimage-tutorials/lib/python3.9/site-packages/jupyter_book/utils.py", line 48, in _error
    raise kind(box)
RuntimeError: 
===============================================================================

The Table of Contents file is malformed: toc is not a mapping: <class 'list'>
You may need to migrate from the old format, using:

        jupyter-book toc migrate /home/ross/repos/skimage-tutorials/_toc.yml -o /home/ross/repos/skimage-tutorials/_toc.yml

===============================================================================
</pre>
</details>

--- a/_toc.yml
+++ b/_toc.yml
@@ -1,6 +1,7 @@
-- file: lectures/00_images_are_arrays
+format: jb-article
+root: lectures/00_images_are_arrays
+sections:
 - file: lectures/1_image_filters
 - file: lectures/3_morphological_operations
 - file: lectures/4_segmentation
 - file: lectures/three_dimensional_image_processing
-

